todo_app/app.py

```diff
@@ -7,172 +7,177 @@
 from operator import itemgetter
 from todo_app.data.Trello import get_member, hydrate_member, update_task_status, delete_task, add_task
 
-
-app = Flask(__name__)
-app.config.from_object(Config)
-    
-
-def get_last_sort_col():
-
-    lsc = request.args.get('last_sort_col')
-    if not lsc:
-        lsc = session.get('last_sort_col')
-    if not lsc:
-        lsc = 'id'
-    return lsc
-    
-def get_sort_col():
-    sc = request.args.get('sort_col')
-    if not sc:
-        sc = session.get('sort_col')
-    if not sc:
-        sc = "id"
-    return sc
-
-
-def get_sort_dir():
-    sd = request.args.get('sort_dir')
-    if not sd:
-        sd = session.get('sort_dir')
-    if not sd:
-        sd = 'asc'
-    return sd    
-
-def get_preserve_sort():
-    pso = session.get('preserve_sort')
-    session['preserve_sort']=None
-    return pso==True
-
-def get_sort_parameters():
-    
-    if(not request.args.get('sort_col') and not session.get('sort_col')):
-        sort_col='id'
-        last_sort_col=None
-        sort_reverse=False
-        sort_dir='des'
-        return sort_col, last_sort_col, sort_reverse, sort_dir    
-    else:
-        sort_col = get_sort_col()
-        last_sort_col = get_last_sort_col()
-        sort_dir = get_sort_dir()
-        if(not get_preserve_sort()):
-            if (sort_col == last_sort_col):
-                if (sort_dir == 'asc'):
-                    sort_dir='des'
-                else:
-                    sort_dir='asc'    
-        if sort_dir == 'asc':
-            sort_reverse = False
+def create_app():
+
+    app = Flask(__name__)
+    app.config.from_object(Config)
```

Comment:
Currently your integration test fails when I run it from the command line because the SECRET_KEY variable isn't set - that happens because it's checked when the `Config` file is first imported, which occurs _before_ the integration test has time to load the `.env.test` file. There are various ways to solve that (e.g. switching the Config to set SECRET_KEY inside a constructor `init` function) but probably easiest is to switch this to a string with the module name rather than the Config class directly, which will delay when it's first loaded:
```suggestion
    app.config.from_object("todo_app.flask_config.Config")
```

That, in combination with removing the now defunct Config import statement, should allow that test to pass from the terminal again